src/language/ast.js

```diff
@@ -284,6 +286,12 @@ export type BooleanValue = {
   value: boolean;
 }
 
+export type NullValue = {
+  kind: 'NullValue';
+  loc?: Location;
+  // value: null;
```

Comment:
I was in doubt with this. Can I add `value` property? Because other literal AST nodes have `value` - so I keep it commented. So you suggest remove `value` or actually set it to null?

I think remove it will be better, but thank you for your careful review :+1: